plugin/api.go

```diff
@@ -450,6 +450,30 @@ type API interface {
 	// Minimum server version: 5.6
 	SearchChannels(teamID string, term string) ([]*model.Channel, *model.AppError)
 
+	// CreateChannelSidebarCategory creates a new sidebar category for a set of channels.
+	//
+	// @tag Channel
+	// @tag User
+	// @tag Team
```

Comment:
What if we create a new tag, `ChannelSidebar`, instead of adding these three? (same comment for the other two declarations)

Reply:
Sounds good to me, added the new tag ✅ 

Should we explicitly create a new tag or just adding them is sufficient? Also, how are these tags used/parsed?

Reply:
Just adding them is enough. The tags are parsed when building the plugin documentation, which outputs the page in https://developers.mattermost.com/extend/plugins/server/reference/. The code parsing the tags is in https://github.com/mattermost/mattermost-developer-documentation/blob/master/cmd/plugin-godocs/plugin-godocs.go#L73-L81

Reply:
ah I see thank you